Fun Commands/Mini-Games/Dyno-RPG/blight.txt

```diff
@@ -0,0 +1,15 @@
+{delete}
+{silent}
+{require:{&Adventurer}}
```

Comment:
This could be added as a note? Since Dyno's command flags so to speak are now done via the command creation menu.

Reply:
Oh, yeah I add them in for those who don't want to use that lol. But if it's preferred I can change that.

Reply:
aaand fixed